pkg/build/controller/build/build_controller.go

```diff
@@ -285,11 +290,76 @@ func NewBuildController(params *BuildControllerParams) *BuildController {
 	c.imageStreamStoreSynced = params.ImageStreamInformer.Informer().HasSynced
 	c.buildControllerConfigStoreSynced = params.BuildControllerConfigInformer.Informer().HasSynced
 	c.imageConfigStoreSynced = params.ImageConfigInformer.Informer().HasSynced
-	c.configMapStoreSynced = params.OpenshiftConfigConfigMapInformer.Informer().HasSynced
+	c.openshiftConfigConfigMapStoreSynced = params.OpenshiftConfigConfigMapInformer.Informer().HasSynced
+	c.controllerManagerConfigMapStoreSynced = params.ControllerManagerConfigMapInformer.Informer().HasSynced
 
 	return c
 }
 
+// additionalTrustedCAs returns a copy of the additional trusted certificate authorities
+// to be used by the build pod
+func (bc *BuildController) additionalTrustedCAs() map[string]string {
+	bc.configLock.Lock()
+	defer bc.configLock.Unlock()
+	caData := make(map[string]string)
+	if bc.additionalTrustedCAData == nil {
+		return caData
+	}
+	for k, v := range bc.additionalTrustedCAData {
+		caData[k] = v
+	}
+	return caData
+}
+
+func (bc *BuildController) setAdditionalTrustedCas(value map[string]string) {
+	bc.configLock.Lock()
```

Comment:
aren't you only calling this once, during startup?  why do we need locks here and on the read above?

Reply:
nm, i guess you're refreshing this data on every build?

Reply:
or at least when the config object changes?

Reply:
No - we call this every sync for the cluster config.

Reply:
We also run sync every time a ConfigMap in the `openshift-config` namespace changes.